apis/pkg/v1alpha1/config.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ControllerConfigSpec specifies the configuration for a packaged controller.
+// Values provided will override package manager defaults. Labels and
+// annotations are passed to both the controller Deployment and ServiceAccount.
+type ControllerConfigSpec struct {
```

Comment:
@turkenh could you take a look at the configurable fields here and make sure they fit all of your use-cases?

Reply:
yes, contains all fields that we would need to configure 👍